CMakeLists.txt

```diff
@@ -298,6 +299,7 @@ SET(PLUGINS
     # Selectable plugins
 
     dataStore/CSVDataStore
+    dataStore/BSMLDataStore
```

Comment:
You might want to use ```BioSignalMLDataStore``` to be in line with the ```BioSignalMLAPI``` plugin. Also because the format itself is going to be referenced as ```BioSignalML``` in the GUI.